examples/app.py

```diff
@@ -35,22 +35,121 @@
 
 from __future__ import absolute_import, print_function
 
+import os
+from time import sleep
+
 from flask import Flask
 from flask_babelex import Babel
-from flask_mail import Mail
+from flask_menu import Menu as FlaskMenu
+from invenio_access import InvenioAccess
 from invenio_accounts import InvenioAccounts
-from invenio_db import InvenioDB
+from invenio_accounts.testutils import create_test_user
+from invenio_accounts.views import blueprint as blueprint_user
+from invenio_admin import InvenioAdmin
+from invenio_db import InvenioDB, db
+from invenio_indexer import InvenioIndexer
+from invenio_indexer.api import RecordIndexer
+from invenio_mail import InvenioMail as Mail
+from invenio_pidstore import InvenioPIDStore
 from invenio_records import InvenioRecords
+from invenio_records_ui import InvenioRecordsUI
+from invenio_search import InvenioSearch
+
 from zenodo_accessrequests import ZenodoAccessRequests
+from zenodo_accessrequests.views.requests import blueprint as request_blueprint
+from zenodo_accessrequests.views.settings import \
+    blueprint as settings_blueprint
 
 # Create Flask application
 app = Flask(__name__)
+
+app.config.update(
+    #  DEBUG=True,
+    CELERY_ALWAYS_EAGER=True,
+    CELERY_CACHE_BACKEND="memory",
+    CELERY_EAGER_PROPAGATES_EXCEPTIONS=True,
+    CELERY_RESULT_BACKEND="cache",
+    MAIL_SUPPRESS_SEND=True,
+    TESTING=True,
+    SECRET_KEY='TEST',
+    SQLALCHEMY_DATABASE_URI=os.environ.get(
+        'SQLALCHEMY_DATABASE_URI', 'sqlite:///app.db'
+    ),
+    SECURITY_PASSWORD_SALT='security-password-salt',
+    RECORDS_UI_ENDPOINTS=dict(
```

Comment:
Can you move the ``recid_access_request`` and ``recid_accss_request_email_confirm`` to ``zenodo_accessrequests/config.py``. This way it's easier to use:

```python
#zenodo_accessrequests/config.py
ACCESS_REQUESTS_RECORDS_UI_ENDPOINTS=dict(
    recid_access_request=dict(
        pid_type='recid',
        route='/records/<pid_value>/accessrequest',
        template='zenodo_accessrequests/access_request.html',
        view_imp='zenodo_accessrequests.views.requests.access_request',
        methods=['GET', 'POST'],
    ),
    recid_access_request_email_confirm=dict(
        pid_type='recid',
        route='/records/<pid_value>/accessrequest/<token>/confirm',
        #  template='invenio_records_ui/detail.html',
        view_imp='zenodo_accessrequests.views.requests.confirm',
    ),
)
```

```python
# zenodo/config.py
from zenodo_accessrequests import ACCESS_REQUESTS_RECORDS_UI_ENDPOINTS
RECORDS_UI_ENDPOINTS.update(ACCESS_REQUESTS_RECORDS_UI_ENDPOINTS)
```